main.go

```diff
@@ -37,22 +37,29 @@ func main() {
 
 	var tags []string
 
-	switch repo {
-	case "hub.docker.com":
+	switch {
+	case strings.Contains(repo, "hub.docker.com"):
```

Comment:
This is expected to match only `hub.docker.com`, but actual matches also `*.hub.docker.com` or `hub.docker.com.*`.

```suggestion
	case repo == "hub.docker.com":
```

Reply:
fixed in 557f9da